src/isyntax/isyntax.c

```diff
@@ -1817,95 +1816,102 @@ u32* isyntax_load_tile(isyntax_t* isyntax, isyntax_image_t* wsi, i32 scale, i32
 			} break;
 		}
 
-		// Distribute result to child tiles
-		if (scale > 0) {
-			isyntax_level_t* next_level = wsi->levels + (scale - 1);
-			isyntax_tile_t* child_top_left = next_level->tiles + (tile_y*2) * next_level->width_in_tiles + (tile_x*2);
-			isyntax_tile_t* child_top_right = child_top_left + 1;
-			isyntax_tile_t* child_bottom_left = child_top_left + next_level->width_in_tiles;
-			isyntax_tile_t* child_bottom_right = child_bottom_left + 1;
-
-			ASSERT(child_top_left->color_channels[color].coeff_ll == NULL);
```

Comment:
Note actual logic change below, as I now allow tiles to have coefficients that may need to be released. This is because some tiles may be evicted and some not, so I can't expect them all to be either null or non-null.